app/views/assessments/_partial_feedback_js.html.erb

```diff
@@ -1,22 +1,33 @@
 <%= external_javascript_include_tag "lodash" %>
 <script>
+  const sessionKeyJson = {
+    submission: <%= @submission_id %>,
+    job: <%= @job_id %>,
+    user: <%= @cud&.user_id %>,
+  }
+  const sessionKey = JSON.stringify(sessionKeyJson);
+  
+  function loadPartialFeedback(partialFeedback) {
+    $('#partial-feedback').html(`
+      <pre>${partialFeedback}</pre>
+      <div class="autograding-in-progress">
+        <i class="refresh-feedback small material-icons" onclick="rotateIcon(); onClickRefresh()">refresh</i>
+        <pre>Autograding in progress</pre>
+      </div>`);
+    $('.feedback-status').html("In-Progress");
+    $('.feedback-status').removeClass("feedback-status__queued");
+    $('.feedback-status').addClass("feedback-status__inprogress");
+    $( "#result-body" ).load(window.location.href + " #result-body" );
```

Comment:
May I ask what's going on in this line of code? I don't quite get what this line does.

Reply:
Updates the red "in-progress" badge into a blue "queued" badge 
![image](https://user-images.githubusercontent.com/50491000/208901602-d7b443d8-9214-4ca3-a8c0-38da07e3a477.png)